templates/layout/base.html

```diff
@@ -11,12 +11,21 @@
         <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
 
         <script>
-            // Change document class from no-js to js so we can detect this in css
+            {{!-- Change document class from no-js to js so we can detect this in css --}}
             document.documentElement.className = document.documentElement.className.replace('no-js', 'js');
         </script>
 
         {{ getFontsCollection }}
         {{{stylesheet '/assets/css/theme.css'}}}
+        {{#unless theme_settings.lazyload '==' 'disabled'}}
+            {{!-- Load Lazysizes script ASAP so images will appear --}}
+            <script>
+                {{!-- Only load visible elements until the onload event fires, after which preload nearby elements. --}}
+                window.lazySizesConfig = window.lazySizesConfig || {};
+                window.lazySizesConfig.loadMode = 1;
+            </script>
+            <script src="{{cdn 'lazysizes.min.js'}}"></script>
```

Comment:
@junedkazi @mattolson curious what you think about this.

If we wait until the bundle loads for lazysizes to activate, then no images will show up until the bundle executes.

Any scripts in `head.scripts` will be *effectively* blocking at that point, because they're preventing images from loading.

If we make lazysizes load before head.scripts, we have insurance against this - but I just wanted to get your thoughts on pulling this out of the bundle in general, since we don't typically do that. Maybe there's a more "webpack-ey" way to do this?